README.md

```diff
@@ -1 +1,188 @@
-# Budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget App ](#-Budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+
+# 📖 Budget App <a name="about-project"></a>
+
+Budget App is a web application that enables users to create, share, and explore Budgets and Transactions. Users can effortlessly create their own Budgets, including detailed category lists, transactions/entities while setting and also viewing of the prices of the entities are also made possible
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Project language </summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>FrameWork</summary>
+  <ul>
+    <li><a href="">ROR</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>DataBase</summary>
+  <ul>
+    <li><a href="">Postgres</a></li>
+  </ul>
+</details>
+
+### Live Demo
+
+- **https://crib-zoil.onrender.com/**
+
+### Loom Presentation
+
+**https://www.loom.com/share/163c4f210f5e48f7b00f0772c90496fa**
```

Comment:
- [ ] Good job presenting your project but you forgot to add description to a piece of code you are proud of